extensions/search-composer-packagist/src/composer.tsx

```diff
@@ -0,0 +1,105 @@
+import { ActionPanel, CopyToClipboardAction, List, OpenInBrowserAction, showToast, ToastStyle } from "@raycast/api";
+import { useEffect, useMemo, useState } from "react";
+import algoliaSearch from "algoliasearch/lite";
+
+const APPID = "M58222SH95";
+const APIKEY = "5ae4d03c98685bd7364c2e0fd819af05";
+const INDEX = "packagist";
+
+type PackagistHitMeta = {
+  downloads: number;
+  downloads_formatted: string;
+  favers: number;
+  favers_formatted: string;
+};
+
+type PackagistHit = {
+  id: number;
+  name: string;
+  package_organisation: string;
+  package_name: string;
+  description: string;
+  type: string;
+  repository: string;
+  language: string;
+  trendiness: number;
+  popularity: number;
+  meta: PackagistHitMeta;
+  tags: string[];
+};
+
+function getComposerRequireCommand(hit: PackagistHit) {
+  return `composer require ${hit.name}`;
+}
+
+function getPackagistPageURL(hit: PackagistHit) {
+  return `https://packagist.org/packages/${hit.name}`;
+}
+
+export default function SearchDocumentation() {
+  const algoliaClient = useMemo(() => {
+    return algoliaSearch(APPID, APIKEY);
+  }, [APPID, APIKEY]);
+
+  const algoliaIndex = useMemo(() => {
+    return algoliaClient.initIndex(INDEX);
+  }, [algoliaClient, INDEX]);
+
+  const [searchResults, setSearchResults] = useState<any[] | undefined>();
+  const [isLoading, setIsLoading] = useState(false);
+
+  const search = async (query = "") => {
+    if (query === "") {
+      return;
```

Comment:
I think we should return and empty array here also, to be consistent in our return values.